src/classes/GE_PaymentServices.cls

```diff
@@ -83,6 +94,16 @@ public with sharing class GE_PaymentServices {
         config = new Configuration().keyValueMap;
     }
 
+    @AuraEnabled(cacheable=true)
+    public static String getPaymentTransactionStatus() {
```

Comment:
See other comment about rename to "getPaymentTransactionStatusEnum" or "getPaymentTransactionStatusValues"